spec/mailers/track_mailer_spec.rb

```diff
@@ -53,9 +53,20 @@
 
       it 'should update the time of the user\'s last daily tracking email' do
         expect(@user).to receive(:last_daily_track_email=).with(Time.zone.now)
-        expect(@user).to receive(:save!)
+        expect(@user).to receive(:save!).with(touch: false)
         TrackMailer.alert_tracks
       end
+
+      it 'should not update the time of the user was last updated' do
```

Comment:
```suggestion
      it 'should not update the user\'s last updated timestamp' do
```

Reply:
I'd go with:

```ruby
it "does not update the user's last updated timestamp" do
```

* http://www.betterspecs.org/#should
* https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals-double-quote